Readme.md

```diff
@@ -8,3 +8,32 @@ The new cardano-explorer consists of three components:
   date from the chain into a PostgreSQL database.
 * `cardano-explorer` which serves data from the PostgreSQL database via HTTP.
 
+
+## Schema Management
+
+Schema management for the Cardano Explorer database is a little complicated than we would like,
```

Comment:
Only a minor nit but that would read better for me as:

"a little more complicated".